app/containers/MessageBox/index.js

```diff
@@ -583,12 +584,17 @@ class MessageBox extends Component {
 	}
 
 	showFileActions = () => {
-		ActionSheet.showActionSheetWithOptions({
-			options: this.fileOptions,
-			cancelButtonIndex: FILE_CANCEL_INDEX
-		}, (actionIndex) => {
-			this.handleFileActionPress(actionIndex);
-		});
+		const { FileUpload_Enabled } = this.props;
+		if (FileUpload_Enabled) {
```

Comment:
Instead of showing an alert, we use to hide the feature (the button).
Can you change it?

Reply:
Sorry for my slow reply - yes I can however would you allow this as the next PR?

Reply:
@grahamsmith feel free to do it on this same PR, just check if `File Upload` is `enabled` before show the file upload button. 😄